shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
+
+class ShopListRepository(private val shopListApi: ShopListApi) {
```

Comment:
there is no domain layer. its better to have a interface called ShopListRepository inside the repository list folder.
in the network data layer you can use implementation class of it "ShopListRepositoryIml"